src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/NodeHardwareScaler/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry title="vCPU(s) Per Executor" field="vCpuPerExecutor">
+        <f:number clazz="required number" default="0"/>
```

Comment:
I think we can use `clazz="number"` here.

Reply:
Should this not be a required field if Scaling by Node Hardware is chosen?